Source/Core/Core/ConfigManager.cpp

```diff
@@ -613,6 +614,7 @@ void SConfig::LoadMovieSettings(IniFile& ini)
   movie->Get("PauseMovie", &m_PauseMovie, false);
   movie->Get("Author", &m_strMovieAuthor, "");
   movie->Get("DumpFrames", &m_DumpFrames, false);
+  movie->Get("DumpFramesBitrate", &m_DumpFramesBitrate, 2500);
```

Comment:
`DumpFramesBitrateKbps` would be more self-documenting with regards to what the representation of the value is.